torch/utils/tensorboard/writer.py

```diff
@@ -505,8 +505,8 @@ def add_image(self, tag, img_tensor, global_step=None, walltime=None, dataformat
         Shape:
             img_tensor: Default is :math:`(3, H, W)`. You can use ``torchvision.utils.make_grid()`` to
             convert a batch of tensor into 3xHxW format or call ``add_images`` and let us do the job.
-            Tensor with :math:`(1, H, W)`, :math:`(H, W)`, :math:`(H, W, 3)` is also suitible as long as
-            corresponding ``dataformats`` argument is passed. e.g. CHW, HWC, HW.
+            Tensor with :math:`(1, H, W)`, :math:`(H, W)`, :math:`(H, W, 3)` is also suitable as long as
+            corresponding ``dataformats`` argument is passed. e.g. ``CHW``, ``HWC``, ``HW``.
```

Comment:
Could you add a comma after `...is passed.` instead of a fullstop?

Reply:
Sure, that'll help with clarity.

Reply:
@vishwakftw I made that change in c1b0501.  At first I misinterpreted - the a92b370 commit was a mistake.